variables.tf

```
@@ -80,3 +79,8 @@ variable "name_prefix" {
   type        = string
   description = "A prefix to add to the names of all created resources."
 }
+
+variable "interface_endpoint_ingress_sg" {
+  type        = string
+  description = "Security groups ingress to associate with the interface endpoint"
```

Unless we make this a list of strings, this only supports a single security group ID. That being said, should we be supporting a list of security groups? Like for the Tamr VM and the EMR cluster?
```suggestion
  description = "Security group ID ingress to associate with the interface endpoint"
```

---

For the use that we are going to give, having the variable being a string is sufficient. This variable description could be re-worded to:
**description** = "Application Security group ingress ID (aka Tamr VM security group ID) to associate with the interface endpoint".